src/pages/documentation/release-notes/index.md

```diff
@@ -6,6 +6,14 @@ description: Release notes and change logs for the Adobe Experience Platform Mob
 
 ## Aug 8, 2023
 
+### iOS Places 4.1.0 and Android Places 2.1.0
```

Comment:
Should we split these in two sections since iOS was released on Aug 7th? 

Also don't forget to also update https://developer.adobe.com/client-sdks/documentation/places/release-notes/ with the same notes.

Reply:
I updated them in the Places release notes as well.
Regarding the dates, I thinking not splitting and keep the same for iOS and Android for easier reading and non-duplication